features/support/api_helpers.rb

```diff
@@ -95,7 +95,7 @@ def ensure_no_framework_agreements_exist(framework_slug)
 
 def update_framework_agreement_status(framework_slug, supplier_id, status)
   response = call_api(:post, "/suppliers/#{supplier_id}/frameworks/#{framework_slug}", payload: {
-    frameworkInterest: {agreementReturned: status},
+    frameworkInterest: {onFramework: status},
```

Comment:
Looks like this method is no longer doing what it says?  i.e. it's called `update_framework_agreement_status` but it actually flips whether the supplier is on the framework or not, and doesn't touch the agreement status at all?